pandas/_libs/hashtable.pyi

```diff
@@ -196,15 +196,21 @@ class HashTable:
         *,
         return_inverse: Literal[True],
         mask: None = ...,
-    ) -> tuple[np.ndarray, npt.NDArray[np.intp],]: ...  # np.ndarray[subclass-specific]
+    ) -> tuple[
+        np.ndarray,
+        npt.NDArray[np.intp],
```

Comment:
Does ruff split this up in multiple rows because there is a trailing comma `npt.NDArray[np.intp],]`? Wouldn't mind removing that comma.

Reply:
Looks like it. I'll remove the comma